frontend/Dockerfile

```
@@ -1,12 +1,14 @@
-FROM node:12.2.0-stretch-slim
+FROM node:current-stretch-slim
```

For Docker images, using anything other than a specific, carefully selected and tested tagged image is considered to be a bad practice - we want to set up Continous Delivery, so when it comes to builds, we have to minimize the risk and disallow third parties from introducing updates that may unexpectedly break backwards compatiblity. This is even more true for the *Dockerfile.production* image.

**TLDR** version numbers should be bumped manually.

```suggestion
FROM node:12.6.0-stretch-slim
```